### ISSUE TITLE
[FEATURE] Define specific value for button pressed and released

### ISSUE THREAD

**Issue by TimSusa** [TimSusa/midi-sliders#44]

**Comment by TimSusa** [comment]
CC buttons: ON = 127, OFF = 0
NOTE buttons (via velocity): ON = vel 127, OFF vel = 0

**Comment by alectroniq** [comment]
i guess a big practical problem is related to this, which is a blocker for me. but I'm not sure, if the problem is in MIDI BRICKS or in REASON (which is my use case): 

a BUTTON_TOGGLE_CC seems to command the assigned DAW button: 
"whatever your state is, just **invert** it"
because randomly there it has a reversed behaviour: I activate a MIDI BRICKS button and the assigned DAW button gets deactivated (and vice versa). since hours I'm trying to reproduce any clear circumstance, but it just seems to happen randomly. 

I think, that as soon as I'm able to define values (like in the first comment of this issue), the DAW's interpretation should be clear:
CC<63 means OFF
CC>64 means ON
or
Note Velocity =0 means OFF
Note Velocity >0 means ON

By the way I had the same issue with my other old software "Emulator 2" that we're trying to compete here. I hope we can solve the problem with defineable ON/OFF values!

**Comment by TimSusa** [comment]
I mean you see midi what is coming from midi-bricks, when using midiox or not? There you should observe the values sending to reason. I would claim, midi-bricks will always end ever send the right values. Anyway, I can come up with that feature at next.

**Comment by TimSusa** [comment]
Do you use the listenToCC Functionality? Maybe the event is sent from reason and toggles to the buttons activated state in midi bricks?

**Comment by TimSusa** [comment]
Adressed by: https://github.com/TimSusa/midi-sliders/commit/bab1847f8931fb484e9129f3aeb0d537dc112305

**Comment by alectroniq** [comment]
- this works, but only after defining the values. 
as you already know, the values now sent out by my pre-built preset are:
Note Buttons: 50=OFF, 127=ON
CC Buttons: 0=OFF, 0=ON

- no listener set (Reason isn't directly able to output CC values for MIDI feedback)

for the inverted toggle problem mentioned above I'll open another issue.
